core/src/main/scala/org/broadinstitute/dsde/rawls/entities/local/LocalEntityExpressionQueries.scala

```diff
@@ -85,8 +85,9 @@ trait LocalEntityExpressionQueries {
         attribute <- entityAttributeShardQuery(context.workspaceContext) if entity.id === attribute.ownerId && attribute.name === attrName.name && attribute.namespace === attrName.namespace
       } yield (rootEntityName, entity.name, attribute)
 
+      // TODO discuss this change
       val attributeForNameQuery =
-        if (attrName.namespace.equalsIgnoreCase(AttributeName.defaultNamespace) && attrName.name.toLowerCase.endsWith(Attributable.entityIdAttributeSuffix)) {
```

Comment:
This change is making the reserved names case sensitive. For example with 'name' vs 'NAME', only 'name' will now be reserved.